client/src/components/common/applicant-info-3.vue

```diff
@@ -1,11 +1,9 @@
 <template>
-  <v-form v-model="isValid" ref="step2" id="applicant-info-3-form">
-    <v-container fluid class="pa-0">
+  <v-form v-model="isValid" ref="step3" id="applicant-info-3-form">
```

Comment:
This is actually an "alternate" step 2. (You get this page or applicant-info-2.)

I replaced all the places I could find this, which was just in this file actually.